Allow to move clock pins (#1172)
thewca/wca-regulations: wca-regulations.md

## <article-F><clock><clocksolving> Article F: Clock Solving

- F1) Standard speed solving procedures are followed, as described in [Article A](regulations:article:A) (Speed Solving). Additional regulations that supersede the corresponding procedures in [Article A](regulations:article:A) are described below.
- F2) The judge places the scrambled puzzle onto the mat in any position.
    - F2a) The organization team may enforce using a stand for the puzzle to prevent it from falling before the start of the attempt. If such stands are used, the organization team should make an announcement before the round starts.
        - F2a1) The judge must remove the stand from the mat immediately after the competitor removes the puzzle from it.
        - F2a2) The competitor may change the state of the pins during the inspection. These changes are not counted as moves.
        - F2a3) The competitor must not change the state of any inner clock faces in the puzzle during the inspection. Penalty: disqualification of the attempt (DNF).

> - F3) At the end of the inspection period, the competitor places the puzzle in a standing position. Penalty: disqualification of the attempt (DNF).

Author: ```suggestion
- F3) At the end of the inspection period, the competitor places the puzzle in any position.
```

Reviewer: I think it's orientation not position (location) that needs to be emphasised ('standing position' is a matter of orientation). And a penalty for placing the puzzle not on the mat would still apply. Making these updates would just lead this regulation becoming a copy of A3d.  So to avoid redundancy this reg could be removed and temporarily replaced with a reminder guideline like the proposed 12g2+.

Author: That makes sense, thanks Oliver!

```suggestion
```
